### ISSUE TITLE
Consider changing package to non-copyleft license

### ISSUE THREAD

**Issue by bitc** [bluepeppers/DigitalOcean#4]
First of all, thank you greatly to the author of this package for creating and releasing it. Out of the several other digitalocean packages, this one has the most features, and in my opinion the most well-designed API.

The license of this library is currently AGPL-3. While this is a fine license, it has a certain political message and severely limits the circumstances where this library can be used.

I am sure that the author is aware of all this, and if the AGPL-3 license was deliberately chosen, then I suppose that this issue can be closed.

However, in order to allow this library to be widely used, and also to be used as a dependency of other open source projects, I respectfully request to consider changing the license.

For Haskell libraries on hackage, the most popular license is BSD3 [1], and I think it would be a good choice also for this library (alternatively, MIT would be an equally good choice).

Thank you

[1] https://gist.github.com/beastaugh/943054#file-results-txt

**Comment by lclarkmichalek** [comment]
Yeah, I've always very much been of the opinion 'AGPL-3 until someone asks for BSD, and then I'll change to BSD', so congrats on fulfilling that prophecy 👍 I don't suppose I could be a pain and ask you to make a PR with the license changed?

**Comment by lclarkmichalek** [comment]
Also you like it? Awesome! I always felt that exporting lenses somewhat tainted the library

**Comment by bitc** [comment]
Thanks! I personally don't have strong feelings one way or the other regarding the use of lenses. They work fine here =]
I'll prepare a PR :+1:
